db/exc/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .application import ApplicationError
```

Comment:
Please don't use abbreviations. Please can we rename this folder `exceptions` if that's what its for,